server/package.json

```diff
@@ -13,7 +13,9 @@
     "cors": "^2.8.5",
     "dotenv": "^10.0.0",
     "express": "^4.17.1",
-    "pg": "^8.7.1"
+    "pg": "^8.7.1",
+    "sequelize": "^6.6.5",
+    "sequelize-cli": "^6.2.0"
```

Comment:
I think if this is a CLI then it can be moved to devDependencies since we don't need it for Docker.